src/Inkluzitron/Modules/UserModule.cs

```diff
@@ -128,6 +133,12 @@ public async Task ShowUserInfoAsync([Name("uživatel")] IUser user = null)
                 .WithFooter("informace o uživateli")
                 .WithThumbnailUrl("attachment://badges.png");
 
+            if (userDb.BirthdayDate is DateTime birthday)
```

Comment:
Chceme zobrazovat narozeniny? :monkahmmm: Přijde mi to nebezpečný zobrazovat jindy než v den narozenin.

Reply:
Fair point, schovám to.

Reply:
Na druhou stranu, když už ti bot někdy popřeje, tak to bude stejně k dohledání v historii..